README.md

```diff
@@ -414,6 +414,20 @@ We use gunicorn using the `gunicorn_server.py` entrypoint.  You&#39;ll also want
 ZEN_ENV=br ZEN_PROD=1 ./web/gunicorn_server.py
 ```
 
+## Running with docker
+
+```bash
+$ cp .env.example .env
+$ export $(grep -v '^#' .env | xargs)
+$ docker-compose build
+$ docker-compose run --rm harmony_app pip3.7 install -e git+https://github.com/room77/py77.git@de4c39b79f5cf404a720eaeaeed60aeee6cffa1f#egg=pylib
+$ docker-compose run --rm harmony_app pip3.7 install -e git+https://github.com/druid-io/pydruid.git@2420109553812470ed4d91cd0f8d5e70bbf2cfd3#egg=pydruid
+$ docker-compose run --rm harmony_app yarn install --pure-lockfile --frozen-lockfile --production=false --no-cache
+$ docker-compose run --rm harmony_app yarn run build
+$ docker-compose up -d --force-recreate
+$ docker-compose run --rm harmony_app scripts/create_user.py --first_name "Your First Name" --last_name "Your Last Name" --username "username@example.com" --site_admin --password "123456"
```

Comment:
Ideally the creation of the web image can be done purely from the dockerfiles that exist. I have pushed new dockerfiles that we use internally with commit 646794b16235 that might make this easier.